File: RoutesJava/app/controllers/Help.java
package controllers;

import play.*;
import play.mvc.*;
import views.html.*;
import static play.mvc.Results.*;

import play.i18n.Lang;
import play.libs.F.Option;

public class Help {

  public static Result show(String language) {
    return ok(localized.render("faq", Lang.forCode(language)));
  }

}

File: RoutesScala/app/controllers/Help.scala
package controllers

import javax.inject.Inject

import play.api.i18n._
import play.api.mvc._

class Help @Inject() (val messagesApi: MessagesApi, val langs: Langs) extends Controller with Localized {

  def show(language: String) = ActionWithLanguage(language) { implicit request =>
    // Partially applied route so the app can fill in the language parameter
    val route = routes.Help.show _
    Ok(views.html.localized("help", languages, Some(route)))
  }

}
